conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -128,5 +145,20 @@ def display_health_checks(prefix: str, verbose: bool = False) -> None:
     else:
         print(f"{OK_MARK} There are no packages with altered files.\n")
 
+
+def env_txt_check(prefix: str, verbose: bool) -> None:
     present = OK_MARK if check_envs_txt_file(prefix) else X_MARK
-    print(f"3. Environment listed in environments.txt file: {present}\n")
+    print(f"Environment listed in environments.txt file: {present}\n")
+
+
+@hookimpl
+def conda_health_checks():
+    yield CondaHealthCheck(
+        name="Missing Files",
+        action=missing_files,
+    )
+    yield CondaHealthCheck(
+        name="Altered Files",
+        action=altered_files,
+    )
```

Comment:
While black/ruff mostly take care of formatting for us there are a few small tricks we have to getting it to format how we like, trailing commas are one of these tricks. If we have a list of something (e.g., `foo = [1, 2, 3]` or `foo(1, 2, 3)`) we can force black/ruff to split this over several lines of code by including a trailing comma:

<table>
<tr>
<td>

**before**

</td>
<td>

```
foo = [1, 2, 3,]     
```

</td>
<td>

```
foo(1, 2, 3,)     
```

</td>
</tr>
<tr>
<td>

**after**

</td>
<td>

```
foo = [
    1,
    2,
    3,
]
```

</td>
<td>

```
foo(
    1,
    2,
    3,
)
```

</td>
</tr>
</table>

This likewise means that even if the code could fit on one line, a trailing comma will force it over several lines. So to make this pretty and more readable we need to remove the trailing comma:

```suggestion
    yield CondaHealthCheck(name="Missing Files", action=missing_files)
    yield CondaHealthCheck(name="Altered Files", action=altered_files)
```

Reply:
Good to know this!